### DISCUSSION TITLE
revolt

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1410] [Icon request]
cant grab the raw icon file so here is a screenshot instead.

also check out revolt its really good https://revolt.chat/
<img width="168" alt="Screenshot 2022-09-26 at 11 06 10 PM" src="https://user-images.githubusercontent.com/96893300/192445994-ea994352-ca0a-4338-a775-e6884bc5a75a.png">

**Comment by di0xazin-e** [comment]
Hey man! Here is a instant link to the app icon itself if you are not wanting to wait for it up upload to the website. But if you want to download it from the website once it is uploaded, here is my link.

https://imgur.com/a/VJ2GKsQ
https://macosicons.com/#/u/alivat0r
